### ISSUE TITLE
Change unlock password to "unlock passcode"

### ISSUE THREAD

**Issue by stackingsaunter** [getAlby/lightning-browser-extension#2251]
Users confuse unlock password with Alby Account password. This minor copy change from "password" to "passcode" may help a bit pointing out the difference. 

I also prepared a new illustration for onboarding form that will better show how it works:

![image](https://user-images.githubusercontent.com/68239231/226774194-eeaeea4e-4a2f-4817-9255-baaa895823e3.png)

With that change I also think we can improve copy here a bit (I do not think mentioning Alby Account here serves purpose, as users still confuse this and many users will not use Alby Account at all or don't know yet what is it.). See screenshots (note: second version screenshot is based on [new proposed layout](https://github.com/getAlby/lightning-browser-extension/issues/2250), but changes to copy and screenshot can be applied to current layout as well)

Current: 
**Set an unlock password**
This password is used to protect your wallet and provide access to the browser extension. It cannot be reset and is separate from your Alby account password.

<img width="1261" alt="Screenshot 2023-03-22 at 02 05 33" src="https://user-images.githubusercontent.com/68239231/226775513-e62ce1e1-4c54-4bdd-897f-340fab4d82d4.png">

Proposed: 
**Set an unlock code**
This passcode will allow you to unlock Alby on this device.  It cannot recovered, but may be changed later.

<img width="734" alt="Screenshot 2023-03-22 at 02 07 29" src="https://user-images.githubusercontent.com/68239231/226775751-e4f3996b-abd9-4340-92ed-bb3587c8181d.png">

**Comment by nishtha1425** [comment]
Hello, Can I propose a design-based solution for this issue?

**Comment by nishtha1425** [comment]
Although, passcode is a better option to point out the difference from password, in my opinion if we could ask the user to choose a "key" and if we could mention unlock alby or unlock extension, it would make it clearer for the user.

**Comment by stackingsaunter** [comment]
@nishtha1425 We will soon introduce Secret Key (seed phrase) for nostr and other protocol keys, so I'd rather not use key in this context. Do you have some other design in mind or just copy change?

I added "browser extension" to the copy btw

<img width="727" alt="Screenshot 2023-04-03 at 19 56 38" src="https://user-images.githubusercontent.com/68239231/229589185-9e28e6c8-04dc-42a2-80b2-1887d8ad590c.png">

**Comment by nishtha1425** [comment]
Is the confusion only while choosing the unlock passcode or while trying to access the extension?

**Comment by stackingsaunter** [comment]
I think both, depending on a user. The confusion comes from the fact that Alby account users can log in with password, so sometimes they think this screen asks them to use their Alby account password (it also comes down that they also confuse alby web wallet account with the extension)

**Comment by nishtha1425** [comment]
exactly, so that is what i wanted to say that mentioning it in copy is important but other than that when the user has to put in the unlock passcode at that time we can specify "unlock alby extension" instead of "unlock to continue". I am suggesting change in the unlock screen also other than copy change. Like for example when we have to use any payment wallet the passcode to unlock the application is asked before we open the app and then when we have to make any transaction a different password is required. So when we open that app it says "Unlock app".
This the only suggestion i had in mind. 
Thank you

**Comment by stackingsaunter** [comment]
@nishtha1425 I am not sure I understand your suggestions. There's no password when users are making transactions.

They unlock the extension every time they turn on the browsers. They need Alby account password if they want to use it in the extension, but they need to use it one time only. 

If you'd like to propose some particular design changes, [here's a master file of the extension design](https://www.figma.com/file/O49NS4o3IjWwmHvFLncTy6/Alby-Extension-Master-Design?node-id=0%3A1&t=zgYMu5Z8O4e1BxEx-1) that you can copy and play around with!

**Comment by BhuvanAde** [comment]
@stackingsaunter The new designs are excellent! Cheers, Could you try adding an arrow to the next button as a visual cue that suggests a progression or movement forward? It could also be used to fill the negative space.

**Comment by Pratik-A-N** [comment]
We have to change the password to passcode here @stackingsaunter do I understand the solution right ??

**Comment by stackingsaunter** [comment]
@Pratik-A-N yes, 

- [ ] change copy to:

**Set an unlock code**
This passcode will allow you to unlock Alby on this device.  It cannot recovered, but may be changed later.

- [ ] change "unlock password" to "unlock passcode" everywhere in the extension

<img width="467" alt="Screenshot 2023-04-27 at 17 38 40" src="https://user-images.githubusercontent.com/68239231/234913918-cadb91f4-7f08-4dbb-a697-863cca5f94fb.png">

**Comment by Pratik-A-N** [comment]
@stackingsaunter can you assign this issue to me I'm starting to work on it

**Comment by fczuardi** [comment]
Yes, I agree that it is confusing to have 2 different passwords (one to unlock the extension, and one for the Alby Account). A possible analogy is the login PIN on Windows vs the Microsoft Account password, or the unlocking pattern on an Android phone vs the Google Account password.

Using two different words for password (passcode and password) might not IMHO be a clear hint, and it might even be difficult to translate into other languages, I foresee that the translation might be the same word for both cases, or even passcode might be translated as something that accepts only numbers.
